### ISSUE TITLE
Pipeline that emits individual JSON files for each item

### ISSUE THREAD

**Issue by jessemortenson** [openstates/scrapy-test#10]
Our [existing scrapers](https://github.com/openstates/openstates-scrapers) generally output data as a set of JSON files within a directory. The goal for this scrapy test is to add a scrapy pipeline that matches that output format. Scrapy's default "one JSON file with all output" is OK, but we already have some data engineering code that expect the old output style, as it handles processing the data into our operational database. 

### Goals

1. Match the one-JSON-per-item output style of the old scrapers.
2. Support two options for the destination: a) save these JSON files to a local folder, OR b) save those JSON files to an AWS S3 location. This may actually require creating two scrapy pipelines, I'm not sure. But the structure/naming of the files created should be the same scheme whether the destination is a local folder or S3

### Requirements

1. Feel free to re-use existing logic/code to save time
2. Should be implemented as [scrapy item pipeline](https://docs.scrapy.org/en/latest/topics/item-pipeline.html) or two pipelines, so it feels "native" to the scrapy ecosystem
3. The existing code emits these JSON files to a very simple structure, eg `_data/mo/*` where `mo` is the abbreviation representing the political jurisdiction from which we are scraping. We don't need to stick to that. Something that creates new folders by date would be better, because then the user doesn't have to worry about accidentally overwriting the output of a previous scrape (it can be useful to compare them). So a better directory structure would be more like `_data/mo/2023-11-29/1/*` where `2023-11-29` is the current date and `1` represents the first time I've run this scraper today.
4. You don't need to test the S3 integration with our particular bucket, feel free to just create/use your own AWS account within the free tier of S3 usage. We will take care of making sure it integrates with our particular S3 bucket's structure. Just please ensure that the bucket name/path is set by an environment variable we can override (you will see `S3_REALTIME_BASE` in the old/existing code).

## Technical background:

For example, running the [old MO bills scraper](https://github.com/openstates/openstates-scrapers/blob/main/scrapers/mo/bills.py) creates JSON files iwthin a folder called `mo` for Bill, Vote Event, Jurisdiction, and Organization items. Here's a snippet of files from the directory that it produces:

```
-rw-rw-r-- 1 jesse jesse 3.8K Nov 27 22:38 bill_ff7f8de6-8da7-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse 1.8K Nov 27 22:02 bill_ffc98842-8da2-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse 3.2K Nov 27 21:48 bill_ffdba966-8da0-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse 2.3K Nov 27 21:46 jurisdiction_ocd-jurisdiction-country:us-state:mo-government.json
-rw-rw-r-- 1 jesse jesse  178 Nov 27 21:46 organization_ab8b788c-8da0-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  187 Nov 27 21:46 organization_ab8b788d-8da0-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  186 Nov 27 21:46 organization_ab8b788e-8da0-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  533 Nov 27 23:00 vote_event_0015a864-8dab-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  533 Nov 27 23:00 vote_event_0015a865-8dab-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  538 Nov 27 23:00 vote_event_0015a866-8dab-11ee-8c48-031efcb5e369.json
-rw-rw-r-- 1 jesse jesse  533 Nov 27 23:07 vote_event_00176b62-8dac-11ee-8c48-031efcb5e369.json
```

* [The existing output logic is here](https://github.com/openstates/openstates-core/blob/df55000a00759bd26fef6e32f4ce95c20e7f347f/openstates/scrape/base.py#L157) in the `openstates-core` library. 
* [There is a little cleanup logic here that is related](https://github.com/openstates/openstates-core/blob/df55000a00759bd26fef6e32f4ce95c20e7f347f/openstates/cli/update.py#L69)

**Comment by braykuka** [comment]
@jessemortenson I've developed this pipeline but still needs to fix it. BTW could you please review issue #9 and merge it to the main branch? It's useful for me to work #10 and #3. Thanks.

**Comment by jessemortenson** [comment]
Thank you, sorry for the delay. I have merged in #9.
